package.xml

```diff
@@ -19,24 +19,14 @@
 
   <author>Dirk Thomas</author>
 
-  <buildtool_depend>catkin</buildtool_depend>
-
-  <run_depend version_gte="0.2.19">python_qt_binding</run_depend>
-  <run_depend>python-rospkg</run_depend>
-  <run_depend>qt_dotgraph</run_depend>
-  <run_depend>rosgraph</run_depend>
-  <run_depend>rosgraph_msgs</run_depend>
-  <run_depend>roslib</run_depend>
-  <run_depend>rosnode</run_depend>
-  <run_depend>rospy</run_depend>
-  <run_depend>rosservice</run_depend>
-  <run_depend>rostopic</run_depend>
-  <run_depend>rqt_gui</run_depend>
-  <run_depend>rqt_gui_py</run_depend>
+  <exec_depend>ament_index_python</exec_depend>
+  <exec_depend version_gte="0.2.19">python_qt_binding</exec_depend>
```

Comment:
The minimum version doesn't apply for the ROS 2 port and can be removed.

Reply:
Removed